app/components/embed/media/embed_this_form_component.html.erb

```diff
@@ -1,9 +1,23 @@
 <div class='sul-embed-embed-this-form'>
-  <label for="sul-embed-purl-link" class="form-label" readonly>Share link</label>
-  <input type="text" name="sul-embed-purl-link" id='sul-embed-purl-link' value="<%= purl_url %>" readonly>
+  <section data-controller="clipboard">
```

Comment:
Can this be a `div` rather than a section?  A section has semantics, but I don't see them being used here.